### ISSUE TITLE
[🍍]: "getActivePinia()" was called but there was no active Pinia.

### ISSUE THREAD

**Issue by martinszeltins** [CodeDredd/pinia-orm#1585]
### Environment

````
- Operating System: Linux (Ubuntu 20.04 LTS)
- Node Version:     v18.12.1
- Nuxt Version:     3.7.0
- CLI Version:      3.7.3
- Nitro Version:    2.6.2
- Package Manager:  pnpm@8.1.0
- Builder:          -
- User Config:      devtools, modules
- Runtime Modules:  @pinia/nuxt@0.4.11, @pinia-plugin-persistedstate/nuxt@1.1.1, @pinia-orm/nuxt@1.6.7
- Build Modules:    -
````

### Reproduction

https://github.com/martinszeltins/nuxt-pinia-orm

Just run `pnpm install && pnpm run dev`

### Describe the bug

I just started a new Nuxt project, added Pinia ORM and created a model and I am getting this error:

````
[🍍]: "getActivePinia()" was called but there was no active Pinia. Did you forget to install pinia? const pinia = createPinia() app.use(pinia) This will fail in production.

at Query.commit (./node_modules/.pnpm/pinia-orm@1.6.7_pinia@2.1.6/node_modules/pinia-orm/dist/index.js:630:91)
at Query.storeFind (./node_modules/.pnpm/pinia-orm@1.6.7_pinia@2.1.6/node_modules/pinia-orm/dist/index.js:833:23)
at Query.select (./node_modules/.pnpm/pinia-orm@1.6.7_pinia@2.1.6/node_modules/pinia-orm/dist/index.js:897:23)
at Query.internalGet (./node_modules/.pnpm/pinia-orm@1.6.7_pinia@2.1.6/node_modules/pinia-orm/dist/index.js:871:25)
at Query.get (./node_modules/.pnpm/pinia-orm@1.6.7_pinia@2.1.6/node_modules/pinia-orm/dist/index.js:850:19)
at Repository.all (./node_modules/.pnpm/pinia-orm@1.6.7_pinia@2.1.6/node_modules/pinia-orm/dist/index.js:1566:25)
at setup (./app.js:16:28)
at _sfc_main.setup (./app.js:32:23)
at callWithErrorHandling (./node_modules/.pnpm/@vue+runtime-core@3.3.4/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:156:18)
````

All the code is in the [reproduction](https://github.com/martinszeltins/nuxt-pinia-orm) repo, just run `pnpm install && pnpm run dev`

My User model:

```js
import { Model } from 'pinia-orm'
import { Str, Uid } from 'pinia-orm/dist/decorators'

export default class User extends Model {
    static entity = 'users'
    
    @Uid() declare id: string
    @Str('') declare name: string
    @Str('') declare email: string
}
```

My app.vue:

```js
<template>
    <div>
        {{ users }}
    </div>
</template>

<script setup lang="ts">
    import User from './stores/models/user'
    import { useRepo } from 'pinia-orm'

    const userRepo = useRepo(User)
    const users = userRepo.all()
</script>
```

**Comment by CodeDredd** [comment]
Hello @martinszeltins ,
this has to do with the new nuxt version 3.7. I don't know yet what the reason is. But somehow pinia is initlized different.
For know as a workaround pls use nuxt version 3.6.5.

**Comment by CodeDredd** [comment]
related issue: https://github.com/nuxt/nuxt/issues/22918

**Comment by CodeDredd** [comment]
Solved with nuxt 3.7.1
